db/migrate/20170809191203_metrics_sequences.rb

```diff
@@ -0,0 +1,29 @@
+class MetricsSequences < ActiveRecord::Migration[5.0]
+  include MigrationHelper
+  def up
+    change_sequences("metrics", 0..23, false)
+    change_sequences("metric_rollups", 1..12, false)
+  end
+
+  def down
+    change_sequences("metrics", 0..23, true)
+    change_sequences("metric_rollups", 1..12, true)
```

Comment:
I just realized that on the down migration I *think* you also have to update the nextval to the highest id in the table or you'll get duplicates.

Reply:
In the old code, we can not create a record in the old tables.
The sequences will overlap

Reply:
Ahhhh I see.  The only other thing we might want to do is ensure we set the value to the "min value for the region" as opposed to 1.  If that happens automatically, then we are good.